source/get-started/analytical-platform-access.html.md.erb

```diff
@@ -0,0 +1,68 @@
+---
+title: Analytical Platform data access
+weight: 21
+last_reviewed_on: 2022-12-28
+review_in: 6 months
+---
+
+# Analytical Platform data access
+
+Our data is extracted daily into the Analytical Platform, where it is further processed. (For _how_ this is done, please
+refer to [the Integrations page](../integrations.html).)
+
+### Access checklist
+
+1. Ensure you have a GitHub account [with **digital.justice.gov.uk** email and multi-factor authentication](https://user-guidance.services.alpha.mojanalytics.xyz/get-started.html#1-github-account).
+1. Request to add your GitHub account to the [**moj-analytical-services** organisation](https://user-guidance.services.alpha.mojanalytics.xyz/get-started.html#request-an-account-from-the-analytical-platform-team).
+
+### Control Panel user
+
+Once you are in the Analytical Platform organisation:
+
+1. Log in to the [control panel](https://controlpanel.services.analytical-platform.service.justice.gov.uk/), which will be your starting point for all interactions.
+1. Set up [multi-factor authentication](https://user-guidance.services.alpha.mojanalytics.xyz/get-started.html#configure-two-factor-authentication-for-the-analytical-platform).
+1. Ensure you fully log in **at least once**, as it creates your user for the following steps.
+
+If you succeeded, you should see a similar screen:
+![control panel screenshot](images/ap-control-panel.png)
+
+### Production data access
```

Comment:
To add: Development data access

Linking to https://github.com/moj-analytical-services/data-engineering-database-access#standard-database-access

Reply:
done in 63a1fc591dd268f9a516ef71524780e9592ea2fc